test/stylesheet.css

```diff
@@ -0,0 +1,22 @@
+body { font-family: Verdana,sans-serif; }

```

Comment:
This file is not used. Have you forgotten to commit a related test script?
If no related test, could you please move it to the "docs" directory

Reply:
Will move to docs/.